tests/TestHelpers/GraphHelper.php

```diff
@@ -26,16 +26,34 @@ private static function getRequestHeaders(): array {
 		];
 	}
 
+	/**
+	 *
+	 * @return string
+	 */
+	public static function getUUIDv4Regex(): string {
+		return '[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}';
+	}
+
 	/**
 	 * @param string $id
 	 *
 	 * @return int (1 = true | 0 = false)
 	 */
 	public static function isUUIDv4(string $id): int {
-		$regex = '/^[0-9A-F]{8}-[0-9A-F]{4}-4[0-9A-F]{3}-[89AB][0-9A-F]{3}-[0-9A-F]{12}$/i';
+		$regex = "/^" . self::getUUIDv4Regex() . "$/i";
 		return preg_match($regex, $id);
 	}
 
+	/**
+	 * @param string $spaceId
+	 *
+	 * @return int (1 = true | 0 = false)
+	 */
+	public static function isSpaceId(string $spaceId): int {
+		$regex = "/^" . self::getUUIDv4Regex() . '\\$' . self::getUUIDv4Regex() . "$/i";
+		return preg_match($regex, $spaceId);
```

Comment:
```suggestion
		return preg_match($regex, $spaceId) === 1;
```
return bools rather integers, are easier to deal with.

Reply:
Or cast the type as preg_match will return either `0`, `1` or `false` ([preg_match](https://www.php.net/manual/en/function.preg-match.php))
```php
return (bool)preg_match($regex, $spaceId)
```